### PR TITLE
My portfolio Desktop version

### PR DESCRIPTION
**Pull request by Kandy-Peter**
This pull request contains the following elements and steps:

- I created a new branch
- Add HTML/CSS lint
- I used the media query for creating the desktop version 💻 
- Added HTML and CSS files
- Fix linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,142 +62,153 @@ <h2>Glad to see you!</h2>
...
+                    </div>
+                    <p class="card-descript">A daily selection of privately 
+                        personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
                     </ul>
+                    <button>See Project</button>
```
Comment (keneogu) [inline index.html:86]: From the project requirements, when the button is clicked it should take you to the Live Demo of the project or to the GitHub repository. Please fix this in all the buttons.

File: index.html [hunk 2]
```diff
@@ -56,142 +62,153 @@ <h2>Glad to see you!</h2>
...
+                    </li>
+                </ul>
+            </div>
+            <button id="resume-btn">Get my resume</button>
         </div>
-        <button id="resume-btn">Get my resume</button>
+        
         <div class="scroll-menu">
             <ul>
                 <li class="about-list">Languages <i class="fas fa-chevron-right"></i></li>
```
Comment (keneogu) [inline index.html:203]: Please, Add the list of technologies used here. always follow the Figma design been given.
Your design
nn
Figma design
nn1

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Kandy-Peter and @JohnDMurphy, 

Good job so far!

- Linters correctly configured
- Well-written README file.
- Good design

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [x] The `Get in touch` button in the contact me section should be centered.
your design
get
Figma design
get1



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: index.html (hunk 1 above)
Comment (Kandy-Peter) [reply index.html:86]: Hi @keneogu!
we don't have those projects yet and all the buttons are just there for design. I think they will play their roles after completing other projects. The task for today was just to create the mobile and desktop version; it's our first project.
That's why there are no links.

File: index.html (hunk 2 above)
Comment (Kandy-Peter) [reply index.html:203]: As said, we don't need to implement any functionality that requires JavaScript and my last code reviewer agreed that. I didn't think that I should add and we don't yet start with Javascript. Hope, you get me

File: index.html [hunk 3]
```diff
@@ -10,9 +10,15 @@
 </head>
 <body>
```
Comment (MrBrN197) [inline index.html:11]: - [ ]  [OPTIONAL] Hi I can notice that your design is not aligned to the center on the mobile version. Please take the time to ensure that it is aligned in the center in the mobile version.
image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Kandy-Peter  👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**